api/v1/cluster_types.go

```diff
@@ -1162,10 +1162,24 @@ const (
 	SynchronousReplicaConfigurationMethodAny = SynchronousReplicaConfigurationMethod("any")
 )
 
+// DataDurabilityMethod specifies how strictly to enforce synchronous replication
+// when cluster instances are unavailable. Options are `required` or `preferred`.
+type DataDurabilityMethod string
```

Comment:
"method" seems to say, this is the way data durability is achieved, IMO not exactly what we mean.
IMO, perhaps "strictness" or "level" would be clearer
```suggestion
type DataDurabilityLevel string
```

Reply:
There's more info in the issue https://github.com/cloudnative-pg/cloudnative-pg/issues/5793

Reply:
Yes, I read that. But, the "method" we use to get data durability is synchronous replication, no? Here we're just choosing how strict to be in the face of available replicas.

Reply:
I.e. I'd agree with "method" if the alternatives were "strict-synchronous" and "loose-synchronous"

Reply:
Makes sense, I've changed it